src/dashboard/Data/Browser/BrowserTable.react.js

```
@@ -121,10 +122,16 @@
     let editor = null;
     let table = <div ref={this.tableRef} />;
     if (this.props.data) {
-      const rowWidth = this.props.order.reduce(
+      let rowWidth = this.props.order.reduce(
```

ESLint

[error] 'rowWidth' is never reassigned. Use 'const' instead.

(prefer-const)